linux/utils/get_cloudinit_version.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021-2023 VMware, Inc.
```

Comment:
Copyright is 2023 for a new created file.

Reply:
Fixed